spark/v3.5/spark-extensions/src/main/scala/org/apache/spark/sql/catalyst/analysis/RewriteViewCommands.scala

```diff
@@ -40,6 +42,11 @@ case class RewriteViewCommands(spark: SparkSession) extends Rule[LogicalPlan] wi
   override def apply(plan: LogicalPlan): LogicalPlan = plan.resolveOperatorsUp {
     case DropView(ResolvedView(resolved), ifExists) =>
       DropIcebergView(resolved, ifExists)
+
+    case CreateView(r@ResolvedView(_), userSpecifiedColumns, comment, properties,
+    originalText, query, allowExisting, replace) =>
+      CreateIcebergView(r, userSpecifiedColumns, comment, properties, originalText,
```

Comment:
Why not use the `ResolvedIdentifier` here? It looks like other rules expect the `UnresolvedIdentifier` to be replaced with a resolved identifier, which is already produced by `ResolvedView`.